src/js/plugins/plugin.mobile_nav.js

```diff
@@ -0,0 +1,220 @@
+/* global BookReader */
+/**
+ * Adds mobile navigation at responsive breakpoint
+ * NOTE additional script and style tags must be included.
+ *
+ * <script src="../BookReader/mmenu/dist/js/jquery.mmenu.min.js"></script>
+ * <script src="../BookReader/mmenu/dist/addons/navbars/jquery.mmenu.navbars.min.js"></script>
+ * <link rel="stylesheet" href="../BookReader/mmenu/dist/css/jquery.mmenu.css" />
+ * <link rel="stylesheet" href="../BookReader/mmenu/dist/addons/navbars/jquery.mmenu.navbars.css" />
+ */
+
+jQuery.extend(BookReader.defaultOptions, {
+  enableMobileNav: true,
+  mobileNavTitle: 'Internet Archive',
+  mobileNavFullscreenOnly: false,
+});
+
+BookReader.prototype.setup = (function(super_) {
+  return function (options) {
+    super_.call(this, options);
+
+    this.enableMobileNav = options.enableMobileNav;
+    this.mobileNavTitle = options.mobileNavTitle;
+    this.mobileNavFullscreenOnly = options.mobileNavFullscreenOnly;
+
+    this.refs.$mmenu = null;
+  };
+})(BookReader.prototype.setup);
+
+
+// Extend initToolbar
+BookReader.prototype.initToolbar = (function (super_) {
+  return function (mode, ui) {
+    let $mmenuEl;
+
+    if (this.enableMobileNav) {
+      let $drawerEl = this.buildMobileDrawerElement();
+      this.refs.$br.append($drawerEl);
+
+      // Render info into mobile info before mmenu
+      this.buildInfoDiv(this.$('.BRmobileInfo'));
+      this.buildShareDiv(this.$('.BRmobileShare'));
+
+      $mmenuEl = $drawerEl;
```

Comment:
Why not define it here?
```suggestion
      const $mmenuEl = $drawerEl;
```

Reply:
quite interesting find. on line 99 it is referenced and when we define inside of the if block, it doesn't get hoisted up to the top like a `var` does